runtime/src/snapshot_utils.rs

```diff
@@ -1282,6 +1282,19 @@ pub fn purge_old_snapshot_archives<P>(
     snapshot_archives.sort_unstable();
     snapshot_archives.reverse();
     let max_snaps = max(1, maximum_full_snapshot_archives_to_retain); // Always keep at least one snapshot
+    if snapshot_archives.len() > max_snaps {
+        info!(
+            "There are {} full snapshot archives, purging {} of them",
+            snapshot_archives.len(),
+            snapshot_archives.len() - max_snaps
+        );
+    } else {
+        info!(
+            "There are {} full snapshot archives",
+            snapshot_archives.len()
+        );
+    }
```

Comment:
For example, here this could be:

```suggestion
    trace!(
        "There are {} full snapshot archives, purging {} of them",
        snapshot_archives.len(),
        snapshot_archives.len().saturating_sub(max_snaps)
    );
```